src/CppAst.Tests/CppAst.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Lame question, why can't we ugrade in Helios instead? Does it matter if we move to 7.0 or 6.0? From my perspective if we are ugprading we should move to the latest possible

Reply:
I would say we should stick to 6.0, as 6.0 is an LTS version. What worries me is that both 6.0 and 7.0 requires Visual Studio 2022, which then requires Clang 16 which is newer than what CppAST is currently using and has some breaking changes.

I guess in the future we will move up to VS 2022 so these issues have to be resolved, but for now I would avoid making risky moves.

Reply:
Okey-dokey, understood